javascript.js

```diff
@@ -0,0 +1,11 @@
+const menuBtn = document.getElementById('menu-btn');
+const menuList = document.getElementById('menu-list');
+const menuclose = document.getElementById('menu-close-btn');
+
+menuBtn.addEventListener('click', () => {
+  menuList.style.display = 'flex';
+});
+
+menuclose.addEventListener('click', () => {
+  menuList.style.display = 'none';
+});
```

Comment:
- [ ] Good job adding the `href`, but please make sure that, as per requirement, when a user clicks a menu option, the menu disappears. Currently, it stays active even after clicking on the option and being redirected to the correspondant section.

Reply:
thank you, 
I already do it put my mistake😑
 I forget to push before ask for another review.
I push the changes now.